plugins/doc_fragments/module_privatekey_convert.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2016, Yanis Guenane <yanis+ansible@guenane.org>
```

Comment:
Did he really write this? Also applicable for the other copyright lines in this PR.

Reply:
I'm really bad at replacing these things after copying......

Reply:
Fixed in 9b9b759cf2dd8940574a9e173f0c98081b1b6574.